docs/get_vsphere_vm_snapshot.md

```diff
@@ -0,0 +1,30 @@
+# get_vsphere_vm_snapshot
+
+Retrieve summary information for the snapshots of a virtual machine.
+```py
+def get_vsphere_vm_snapshot(self, id, timeout=15):
+```
+
+## Arguments
+| Name        | Type | Description                                                                 | Choices |
+|-------------|------|-----------------------------------------------------------------------------|---------|
+| id  | str  | ID of the virtual machine.    |         |
```

Comment:
Same thing as above. Can we avoid the usage of the ID as the primary search option? Ideally we look this up for the customer dynamically. Probably a little more complicated with the snapshots though.